examples/interactions/injections.py

```diff
@@ -4,10 +4,10 @@
 from dataclasses import dataclass
 from typing import Any, Literal, Optional
 
+import disnake
+
 # this file uses pytz in one of its examples but it is completely optional
 import pytz
-
-import disnake
```

Comment:
Yeah, I'm mainly surprised about the spacing. It threw me off a bit given that this instance here seemed fine, and all three imports are visually grouped together as you'd expect:
https://github.com/DisnakeDev/disnake/blob/cc750bb0a667a9621e3d1233c1cb0ac6cec22f3e/examples/basic_voice.py#L14-L16

Reply:
This is likely because in your example, the comment "trails" at the end of line, but here, the comment is on it's own line. I'll ask on Astral's Discord though to be sure.

Reply:
Looks good, especially if we consider an isolated example:
```py
import os

# Some explanation
import third_party2

import third_party1
```
==>
```py
import os

import third_party1

# Some explanation
import third_party2
```